test/DynamicDependency/Powershell/Test-PackageDependency.Tests.ps1

```diff
@@ -0,0 +1,285 @@
+# Copyright (c) Microsoft Corporation and Contributors.
+# Licensed under the MIT License.
+
+Set-StrictMode -Version 3.0
+
+$ErrorActionPreference = "Stop"
+
+# Repository paths
+$root = (Get-Item $PSScriptRoot).parent.parent.parent.FullName
+$dev = Join-Path $root 'dev'
+$test = Join-Path $root 'test'
+$buildOutput = Join-Path $root 'BuildOutput\Debug\x64'
+$cmdlet = Join-Path $dev 'DynamicDependency\Powershell'
+
+# Test package(s)
+$packageName = 'WindowsAppRuntime.Test.DynDep.Fwk.Widgets'
+$packageFamilyName = 'WindowsAppRuntime.Test.DynDep.Fwk.Widgets_8wekyb3d8bbwe'
+$packageFullName = 'WindowsAppRuntime.Test.DynDep.Fwk.Widgets_1.2.3.4_neutral__8wekyb3d8bbwe'
+$packageMsixDir = 'Framework.Widgets'
+$packageMsixFilename = 'Framework.Widgets.msix'
+$packageMsixPath = Join-Path $buildOutput $packageMsixDir
+$packageMsix = Join-Path $packageMsixPath $packageMsixFilename
```

Comment:
Nit: are these initializations above likely to remain the same as the corresponding ones in Test-MsixDynamicDependency.Test.ps1? If so, consider refactoring to keep only one copy? Just a thought, non blocking.

Reply:
Today yes, but maybe not in time. Didn't see the merit in pulling out a few test data constants into a shared file that would make these otherwise self-contained tests not so. If the complexity grows in time then it may have merit